en/docs/setup/changing-datasource-consent-management.md

```diff
@@ -0,0 +1,248 @@
+# Changing the Default Datasource for Consent Management
+
+WSO2 Identity Server is shipped with an embedded H2 database for storing data. 
+These default databases are located in the `<IS_HOME>/repository/database` directory 
+of the product pack.
+
+By default Consent Management data is stored in the identity database (IDENTITY_DB) with 
+Identity and UMA data. However you can change this to any database type that is 
+supported by WSO2 Identity Server.
+
```

Comment:
However, you can separate consent specific data into a separate data-source to any database type that is 
supported by WSO2 Identity Server.

Reply:
Fixed : 380b00c31c88d7a4de4a940ca60310a3367e5c37